### ISSUE TITLE
Update our domain names for firewalls

### ISSUE THREAD

**Issue by imrehg** [balena-io/docs#993]
We provide wildcard firewall rules in our docs - *.balena-cloud.com, *.docker.io and *.docker.com. Wildcard rules only work with firewalls that parse DNS response packets.

We should as the first step update our docs: show wildcards, but for our own domain also set a date that "As of <date> the full list of subdomains are as follows: ` and add the list (`vpn`, `api`, .... ) Add also caveat that it might change, but we try to keep it static (and that IPs always change, so they cannot have fixed IP)

**Comment by majorz** [comment]
We will also need to list the NTP servers as well - https://github.com/balena-os/meta-balena/blob/master/meta-resin-common/recipes-core/chrony/files/chrony.conf

**Comment by hedss** [comment]
I've actually documented a load of these in the OnPrem guide:
https://github.com/balena-io/balena-on-balena/blob/master/docs/OnPrem/Setup.md#tld-and-dns-entries

I think we need to go through these real-soon-now and discuss this. cc @imrehg and @shaunmulligan to ensure it gets driven.

**Comment by dfunckt** [comment]
This a WONTFIX really -- long story is here: https://jel.ly.fish/238cd5c2-8c5a-45ab-9e77-8ebb68d8759d

tldr; we need to reserve the right to add, remove or update subdomains as we see fit so users must whitelist *.balena-cloud.com.
